### PR TITLE
Add streams-related links for the fetch spec.

### PR DESCRIPTION
**Pull request by yutakahirano**
Needed for https://github.com/whatwg/fetch/pull/119.

### CODE REVIEW

File: xrefs/network/fetch.json [hunk 1]
```diff
@@ -200,7 +215,7 @@
     "responsetype": "responsetype",
     "same-origin data-url flag": "same-origin-data-url-flag",
     "sandboxed-storage-area-urls flag": "sandboxed-storage-area-urls-flag",
-    "should response to request be blocked due to nosniff?": "should-response-to-request-be-blocked-due-to-nosniff?",
```
Comment (annevk) [inline xrefs/network/fetch.json]: Why was this changed?

Comment (yutakahirano) [inline xrefs/network/fetch.json]: I don't know: Just checking out the fetch repository and running anolis make this change, Is my settings wrong?

Comment (annevk) [inline xrefs/network/fetch.json]: Hmm, what does `anolis --version` return?

Comment (yutakahirano) [inline xrefs/network/fetch.json]: anolis 1.3pre

Comment (annevk) [inline xrefs/network/fetch.json]: Weird, I have the same, but mine doesn't output this.

**Comment by yutakahirano** [comment]
@annevk , can you run the following code with python in your environment, and tell me the result?

<pre>
import sys

print('maxunicode = %x' % sys.maxunicode)
</pre>

**Comment by annevk** [comment]
maxunicode = ffff (I'm on OS X).

**Comment by yutakahirano** [comment]
I think I now understand the cause.

See  [utils.py] (https://bitbucket.org/ms2ger/anolis/src/bb7e0d4b84633fa235d804531cac4db4c7469587/anolislib/utils.py?at=default) in anolis source. Between L54-L59 it defines `non_ifragment` regexp. It is used at L107 to replace certain characters with hyphens.

The problem is there are two definitions of `non_ifragment`. `\u003f` a.k.a. `?` is not contained in the first definition (notice the regexp is a positive list) and contained in the second definition (notice the regexp is a negative list). That is why `?` remains in your environment (where maxunicode = 0xfff) and it is removed in my envornment (where maxunicode = 0x10ffff).

I think this is an anolis bug, but it seems [no longer maintained](https://bitbucket.org/ms2ger/anolis/). Maybe we should not use `?`.

**Comment by annevk** [comment]
Nice! Yeah I guess we could revamp `<h4><dfn>Should <var title>response</var> to <var title>request</var> be blocked due to nosniff?</dfn></h4>` somehow.

**Comment by annevk** [comment]
Now that we fixed the "nosniff?" issue, can you rebase this please?

**Comment by yutakahirano** [comment]
Done.
